events/Message/config.js

```diff
@@ -0,0 +1,12 @@
+const { COMMAND_PREFIX } = process.env;
```

Comment:
Por favor, colocar o `"use strict"` nesse arquivo também 💯

Reply:
it's done!

Reply:
Nice!